src/hb-font.cc

```diff
@@ -2236,10 +2236,10 @@ hb_font_get_var_coords_normalized (hb_font_t    *font,
  *
  * Return value: coordinates array
  *
- * Since: REPLACEME
+ * Since: 3.3.0
```

Comment:
This gets filled in at release time based on the chosen version.

Reply:
oh, nice. I should copy that trick